### DISCUSSION TITLE
Home Assistant analytics now show 15% of userbase have installed ZHA integration

### DISCUSSION THREAD

**Discussion by Hedda** [zigpy/zigpy#935] [ZHA development]
FYI, just now noticed that ZHA integration has hit 15% of the Home Assistant userbase and as such made the Top-10 list for the first time!

https://analytics.home-assistant.io/#integrations

I believe it also shows a relativly significant percentage growth as 4-months ago in 16th November 2021 only 13% of the userbase had ZHA:

![image](https://user-images.githubusercontent.com/6320001/158784712-ae2db4a9-87df-4ab3-8057-ae9c6d69c944.png)

I think it would be great if analytics could be extended to also get statistics about radio type and Zigbee devices ZHA users of are using.

I think for zigpy it would be especially interesting to know how large procentage use deconz, znp, zigate, or xbee radio types.

Such data analystics could then maybe be used as statistics to try to convince manufacturers to activly contribute to ZHA and zigpy.

It is worth noting that Home Assistant founders statistics estimate that less than 20% of all users have chosen to electively “opt in” for analytics in Home Assistant to enable Home Assistant founders to collect statistics, (analytics is anonymized before collected of of course):

https://www.home-assistant.io/blog/2021/11/12/100k-analytics/

  “**_Our estimate is that there are 4-5x more installations than people that opt-in to analytics_**.”

If the estimate is still that only 1 out of 5 have choosen to opt-in to analytics then that puts it at around 85,000 estimated ZHA installations.

I think is it also interesting to note that those users who have “opted in” to send analytics today have grown from about 100,000 active Home Assistant installations in November of 2021 to 108,868 active Home Assistant Installations now on the 16th of March 2022.

**Comment by MattWestb** [comment]
2 interesting things is deCONZ is out of the list < 9k hints. Z2M and many more integrations is having 44k hints and likely is around the same as ZHA is taking other MQTT system away (like ESPHome with MQTT, tasmota and so on).

**Reply by Hedda** [reply]
*  deCONZ is currently at 5.7 % which is placed 43 on that integration usage list -> https://analytics.home-assistant.io/#integrations

Zigbee2MQTT usage in Home Assistant is probably not possible to get from their analytics today since abstracted behind MQTT and that can also be by many other integrations and devices as well. [MQTT](https://www.home-assistant.io/integrations/mqtt/) integration is used by 38.7 % (44309) of installations.

Some other integrations for Zigbee capable bridges/hubs/gateway/controllers which at least could be abstracting Zigbee devices;

* [Philips Hue](https://www.home-assistant.io/integrations/hue) at 15.7 % (17970 installs of those who chose to opt-in for Home Assistant Analytics)
* [deCONZ](https://www.home-assistant.io/integrations/deconz) at 5.7 % (6563 installs of those who chose to opt-in for Home Assistant Analytics)
* [IKEA TRÅDFRI](https://www.home-assistant.io/integrations/tradfri) at 3.3 % (3785 installs of those who chose to opt-in for Home Assistant Analytics)
* [Xiaomi Gateway (Aqara)](https://www.home-assistant.io/integrations/xiaomi_aqara) at 2.1 % (2410 installs of those who chose to opt-in for Home Assistant Analytics)
* [Xiaomi Miio](https://www.home-assistant.io/integrations/xiaomi_miio) at 9.5 % (10818 installs of those who chose to opt-in for Home Assistant Analytics)
* [Tuya](https://www.home-assistant.io/integrations/tuya) at 14.9 % (17068 installs of those who chose to opt-in for Home Assistant Analytics)
* [SmartThings](https://www.home-assistant.io/integrations/smartthings) at 3.7 % (4211 installs of those who chose to opt-in for Home Assistant Analytics)
* [Amazon Alexa](https://www.home-assistant.io/integrations/alexa) at 12.6 % (14384 installs of those who chose to opt-in for Home Assistant Analytics)
